lib/perl/Genome/Disk/Allocation.pm

```diff
@@ -605,7 +605,7 @@ sub _execute_system_command {
         my $perl_program_string = join('; ', @statements);
 
         my @cmd = (
-            'genome-perl',
+            $^X,
```

Comment:
I don't know this perl convention and it's not easily googl-able...

Reply:
There's a listing of Perl magic variables here: https://perldoc.perl.org/perlvar.html (I typically search "perlvar" to find it.)

Reply:
`$^X - The name used to execute the current copy of Perl, from C's argv[0] or (where supported) /proc/self/exe.`

TIL